<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TESTOOPClass</class>
 <widget class="QMainWindow" name="TESTOOPClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TESTOOP</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="billlistWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sortpushButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="companylineEdit">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>120</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="calculateTotalpushButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>120</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate total</string>
    </property>
   </widget>
   <widget class="QLabel" name="totalsumlabel">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>160</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>693</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TESTOOP.qrc"/>
 </resources>
 <connections/>
</ui>
